src/fontra/views/editor/editor.js

```diff
@@ -446,7 +446,12 @@ export class EditorController {
 
     const editToolsElement = document.querySelector("#edit-tools");
     const toolButton = html.div(
-      { "class": "tool-button selected", "data-tool": tool.identifier },
+      {
+        "class": "tool-button selected",
+        "data-tool": tool.identifier,
+        "data-tooltip": tool.identifier.replace(/-/g, " "),
```

Comment:
```suggestion
        "data-tooltip": tool.identifier.replaceAll(" ", " "),
```
(The best regex is no regex!)